annotations/static/annotations/js/annotators/text.js

```diff
@@ -327,24 +329,105 @@ DateAppellationCreator = {
         }
     }
 }
+ConceptPickerItem = {
+    props: ['concept'],
+    components: {
+    },
+    template: `<div class="list-group-item concept-item clearfix" id="concept-{{ concept.interpretation.uri }}">
+                <div>
+                    <a v-on:click="select" style="cursor: pointer;">{{ concept.interpretation_label }} ({{ concept.interpretation.authority }})</a>
+                </div>
+                <div class="text text-muted">{{ concept.interpretation.description }}</div>
+            </div>`,
+    methods: {
+        select: function() {
+            this.$emit('selectconcept', this.concept);
+            },
 
+        }
+    }
+
+ConceptPicker = {
+    props: ['appellations'],
+    components: {
+        'concept-picker-item': ConceptPickerItem
+    },
+    data: function() {
+        return {
+            concepts: [],
+            appellationsCopy: [],
+            counts: [],
+            conceptsFinal: [],
+            appell: this.appellations,
+            appellationMap: []
+        }
+    },
+    template: `<div class="appellation-creator" style="max-height: 50vh; overflow-y: scroll;">
+                <concept-picker-item
+                    v-on:selectconcept="selectConcept"
+                    v-for="concept in conceptsFinal"
+                    v-bind:concept=concept>
+                </concept-picker-item>
+               </div>`,
+    methods: {
+        selectConcept: function(concept) {
+            // Clear the concept search results.
+            this.concepts = [];
+            this.$emit('selectconcept', concept);
+        },
+        merge: function (appellations) {
+            this.conceptsFinal = [];
+            this.appell = appellations;
+            this.conceptsFinal = this.appell.slice(0,4)
+            
+            this.appellationMap = this.appellations.map(function (concept, index, array) {
```

Comment:
appellationMap is not actually a map, is it? It's just an array? It shouldn't be called Map if it isn't one.